src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/x64/softmax.cpp

```diff
@@ -0,0 +1,152 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "custom/single_layer_tests/classes/softmax.hpp"
+#include "utils/cpu_test_utils.hpp"
+
+using namespace CPUTestUtils;
+
+namespace ov {
+namespace test {
+namespace SoftMax {
+const auto notOptimizedCPUSpec = CPUSpecificParams{{}, {}, {"ref_any"}, "ref_any"};
+
+const std::vector<SoftMaxConfig> optimizedConfigsFP32 = {
```

Comment:
These `optimized, notOptimized, unsupported` configs were targeting custom modifications of the oneDNN lib. Are those modification still relevant (@luweizhou2016 , @EgorDuplensky)? If not, please remove/join all the shapes configs and modify the common test as well.

Reply:
@luweizhou2016  updated, please help to review

Reply:
The parts changes is okay for me. Thx! Also could you pls help to close the ticket of https://jira.devtools.intel.com/browse/CVS-114609 once this PR is merged?